packages/python-ansible-builder/python-ansible-builder.spec

```diff
@@ -76,6 +73,9 @@ set -ex
 
 
 %changelog
+* Tue Dec 12 2023 Patrick Creech <pcreech@redhat.com> - 1.0.1-7
+- Rollback overzealous obsoletes
```

Comment:
this one contains things in `_bindir` and *needs* the obsoletes.

Reply:
good catch!  I thought I skipped them all, guess not.  Will fix when I come back around to clean up

Reply:
removed